docs/index.md

```diff
@@ -0,0 +1,25 @@
+# iceflow
+
+```{toctree}
+:maxdepth: 2
+:hidden:
+
+```
+
+```{toctree}
+:maxdepth: 2
+:hidden:
+:caption: API
+
+api/*
```

Comment:
TODO: not sure this is working as expected? Or at all? I'd like to have a TOC on the side that has a link to the api docs...